pyi.Dockerfile

```diff
@@ -80,6 +82,23 @@ WORKDIR /tmp
 COPY scripts/burn_build.sh .
 RUN ./burn_build.sh
 
+# node-package-builder-musl
+FROM alpine${NODE_PACKAGE_BUILDER_MUSL} AS node-package-builder-musl
+WORKDIR /tmp
+RUN apk add --no-cache curl g++ python3 make gcc git bash nodejs npm
+COPY scripts/build_node_package.sh .
+RUN ./build_node_package.sh
+
+# node-package-builder-glibc
+FROM ubuntu${NODE_PACKAGE_BUILDER_GLIBC} AS node-package-builder-glibc
+WORKDIR /tmp
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update -y && apt install -y curl g++ python3 make gcc git
```

Comment:
Same as I commented for the alpine part - share it with the `Dockerfile` defintions by creating an `env` script.